### ISSUE TITLE
More visual help to tell different folder levels in explorer

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#8725]
Folders of different levels are hard to tell from one another.

- For example:
 ![image](https://user-images.githubusercontent.com/7269717/192152912-8805c2f1-d3ce-4c63-90f8-883e1ce20348.png)

Currently, the only visual clues are:
1. Directions of the arrow icon
2. Different indent levels.

Of the two, the arrow offers more visual help to tell parents and subfolders but is still hard to tell at a glance.

The Indent levels are barely distinguishable from each other unless one looks very closely, and this minimal difference will disappear when is a distance between to parent levels.

In short, I feel there is a need for settings to add more visual aid to:
1. tell parent from child
2. tell two expanded folders from each other.

Possible aids:
1. Divider between different folders
2. Different shapes or background colors for subfolders of the same parent and for different folders on the same level.
